server/views/pages/componentRequestForm.njk

```diff
@@ -5,238 +5,278 @@
-        text: team.text,
-        selected: team.value == submittedForm.requester_team
-      }), teams) -%}
-    {%- endfor -%}
-    <input type="hidden" name="_csrf" value="{{ csrfToken }}">
-    {{ govukInput({
-      id: "github_repo",
-      name: "github_repo",
-      value: submittedForm.github_repo,
-      classes: "govuk-!-width-three-quarters",
-      label: {
-        text: "Github Repository Name (Should start with (hmpps-), allowed characters alphanumeric and Hyphen(-)) *"
-      },
-      errorMessage: validationErrors | findError('github_repo')
-    }) }}
+      {{ govukSelect({
+        id: "product",
+        name: "product",
+        value: submittedForm.product,
+        label: {
+          text: "Product *",
+          classes: "govuk-fieldset__legend--s"
+        },
+        items: products,
+        errorMessage: validationErrors | findError('product')
+      }) }}
       {{ govukInput({
-      id: "repo_description",
-      name: "repo_description",
-      value: submittedForm.repo_description,
-      label: {
-        text: "Github Repository Description *",
-        classes: "govuk-!-width-three-quarters"
-      },
-      errorMessage: validationErrors | findError('repo_description')
-    }) }}
-    {{ govukRadios({
-      classes: "govuk-radios--inline",
-      idPrefix: "base_template",
-      name: "base_template",
-      value: submittedForm.base_template,
-      errorMessage: validationErrors | findError('base_template'),
-      fieldset: {
-        legend: {
-          text: "Github template repository *",
-          classes: 'govuk-fieldset__legend--s'
+        id: "jira_project_keys",
+        name: "jira_project_keys",
+        value: submittedForm.jira_project_keys,
+        classes: "govuk-!-width-one-third",
+        label: {
+          text: "Jira Project Key",
+          classes: "govuk-fieldset__legend--s"
         }
-      },
-      items: [
-        {
-          value: "hmpps-template-kotlin",
-          text: "hmpps-template-kotlin"
+      }) }}
+      {{ govukInput({
+        id: "github_projects_teams_admin",
+        name: "github_projects_teams_admin",
+        value: submittedForm.github_projects_teams_admin,
+        errorMessage: validationErrors | findError('github_projects_teams_admin'),
+        classes: "govuk-!-width-two-third",
+        label: {
+          text: "Github Repository Admin Teams *",
+          classes: "govuk-fieldset__legend--s"
         },
-        {
-          value: "hmpps-template-typescript",
-          text: "hmpps-template-typescript"
-        }]
-    }) }}
-    {{ govukSelect({
-      id: "github_project_visibility",
-      name: "github_project_visibility",
-      value: submittedForm.github_project_visibility,
-      label: {
-        text: "Github project visbility *"
-      },
-      items: [
-        {
-          value: "public",
-          text: "Public"
+        hint: {
+          text: "Seperated by comma without spaces eg: hmpps-developers,haha"
```

Comment:
little typo on these fields:`separated`

To be robust, we should really handle the possibility of spaces on our side and not worry the user about it. 
`teams.split(",").map(team => team.trim()).filter(team => Boolean(team))`

Which would: 
`"  ,   ,bob,,team  ,  some" => ["bob", "team", "some"]`

Reply:
sorted as suggested